src/components/application_manager/include/application_manager/resumption/resumption_data_db.h

```diff
@@ -65,8 +68,10 @@ class ResumptionDataDB : public ResumptionData {
  public:
   /**
    * @brief Constructor of ResumptionDataDB
+   * @param db_storage show database should be saved in a disk file or in memory
    */
-  ResumptionDataDB();
+  ResumptionDataDB(
+      const application_manager::ApplicationManagerSettings& settings);
```

Comment:
@VVeremjova , this method moved with previous poll-request https://github.com/Kozoriz/sdl_core/blob/feature/Remove_all_singletones/src/components/application_manager/include/application_manager/resumption/resumption_data_db.h#L212